packages/commons/src/auth/jwt.ts

```diff
@@ -52,10 +63,13 @@ export const verifyJwtToken = (jwtToken: string): Promise<boolean> => {
   return new Promise((resolve, _reject) => {
     jwt.verify(jwtToken, getKey(clients), undefined, function (err, _decoded) {
```

Comment:
#424 now validates also the JWT audience, just merged on main - rebase or merge ⚙️